src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -1880,6 +1880,18 @@ static void uct_ib_mlx5_devx_check_xgvmi(uct_ib_mlx5_md_t *md, void *cap_2,
     }
 }
 
+static ucs_status_t
+uct_ib_mlx5_ddp_check_status(uct_ib_mlx5_md_t *md,
+                             ucs_ternary_auto_value_t ddp_enable)
+{
+    if (!(md->flags &
+          (UCT_IB_MLX5_MD_FLAG_DDP_RC | UCT_IB_MLX5_MD_FLAG_DDP_DC))) {
+        return (ddp_enable == UCS_YES) ? UCS_ERR_UNSUPPORTED : UCS_OK;
```

Comment:
shall we use try/yes for force=0/1? forcing means failure is causing error, else ignored.

Reply:
Because this is a cx-8 feature YES => fail if there is no support for ddp at all
No won't even enter the ddp query function and won't set ddp on at all, (ddp is not enabled out of the box)